CHANGELOG.md

```diff
@@ -29,7 +29,12 @@ Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangel
 - **BarSeriesManager** removed empty args constructor
 - **Open|High|Low|Close** do not cache price values anymore
 - **DifferenceIndicator(i1,i2)** replaced by the more flexible CombineIndicator.minus(i1,i2)
-- **DoubleNum** replace redundant `toString()` call in `DoubleNum.valueOf(Number i)` with `i.doubleValue()`
+- **DoubleNum** 
+
+  * replace redundant `toString()` call in `DoubleNum.valueOf(Number i)` with `i.doubleValue()`
+
+  * increases DoubleNum idempotency.
```

Comment:
Please add a more **user friendly description**. What is the main benefit of "DoubleNum idempotency" in compare to the former version?